src/NuGet.Core/NuGet.Commands/MSBuildProjectFactory.cs

```diff
@@ -116,70 +119,53 @@ public PackageBuilder CreateBuilder(string basePath, NuGetVersion version, strin
 
         private void AddOutputFiles(PackageBuilder builder)
         {
-            // Get the target file path
-            foreach (string targetPath in PackTargetArgs.TargetPaths)
+            var allowedOutputExtensions = _allowedOutputExtensions;
+            var listOfFiles = PackTargetArgs.TargetPathsToAssemblies.ToList();
+            if (IncludeSymbols)
             {
-                var allowedOutputExtensions = _allowedOutputExtensions;
+                // Include pdbs for symbol packages
```

Comment:
Mono uses MDB files not PDB files.  Plus the file MDB name format is a little different than PDB's.

For foo.dll and bar.exe:

PDB:  foo.pdb, bar.pdb
MDB:  foo.dll.mdb, bar.exe.mdb

See [this](https://github.com/NuGet/NuGet.Client/pull/905#issue-180346685) for details.